app/views/admin/creator_settings/_form.html.erb

```diff
@@ -52,11 +54,11 @@
     <legend class="crayons-field__label mb-2">Who can join this community?</legend>
     <div>
       <div class="mb-2">
-        <%= radio_button_tag :invite_only_mode, "0", class: "crayons-field crayons-field--radio", required: true %>
+        <%= radio_button_tag :invite_only_mode, "0", false, class: "crayons-radio" %>
```

Comment:
The crayons-radio styles allow the radio buttons to receive the brand color styling. In addition, the required fields weren't being applied here.

Reply:
We added it to honor the positional params for a radio_tag_button:
```
radio_button_tag(name, value, checked, options) 
```

Prior to this change, the class `crayons-field crayons-field--radio` wasn't being applied at all because we were not honouring the position of the arg. However, the radio button was still not being checked because the default for the checked arg is set to false anyways.

Hence, `false` is to indicate that we don't want this radio button to be checked.